src/workerd/util/sqlite.c++

```diff
@@ -236,6 +236,37 @@ static constexpr kj::StringPtr ALLOWED_SQLITE_FUNCTIONS[] = {
   "json_tree"_kj,
 };
 
+// https://www.sqlite.org/pragma.html
+static constexpr kj::StringPtr ALLOWED_READ_PRAGMAS[] = {
+  // We allowlist these SQLite pragmas (for read only, never with arguments).
+  "data_version"_kj,
+  "page_count"_kj,
+  "freelist_count"_kj,
+
+  // Let a user see some internal stats?
+  "max_page_count"_kj,
+  "page_size"_kj,
+};
+
+static constexpr kj::StringPtr ALLOWED_WRITE_PRAGMAS[] = {
+  // We allowlist some SQLite pragmas for changing internal state
+
+  // Toggle constraints on/off
+  "case_sensitive_like"_kj,
+  "foreign_keys"_kj,
+  "defer_foreign_keys"_kj,
+  "ignore_check_constraints"_kj,
+  "recursive_triggers"_kj,
+  "reverse_unordered_selects"_kj,
+
+  // Not "writable", but takes an argument of table name
```

Comment:
Another reason to move away from read/write naming and instead group pragmas by what argument constraints we enforce on them.

Incidentally for these five pragmas below I would like to apply `regulator.isAllowedName`.